code/modules/hallucinations/effects/major.dm

```diff
@@ -0,0 +1,410 @@
+/**
+  * # Hallucination - Terror Infestation
+  *
+  * Creates spider webs and a terror spider near a random vent around the target.
+  */
+/obj/effect/hallucination/terror_infestation
+	duration = 30 SECONDS
+
+/obj/effect/hallucination/terror_infestation/Initialize(mapload, mob/living/carbon/target)
+	. = ..()
+
+	// Find a vent around us
+	var/list/vents = list()
+	for(var/obj/machinery/atmospherics/unary/vent_pump/vent in range(world.view, target))
+		vents += vent
+	if(!length(vents))
+		return
+
+	// Spawn webs around a random vent
+	var/obj/vent = pick(vents)
+	for(var/t in RANGE_TURFS(1, vent))
+		var/turf/T = t
+		if(!isfloorturf(T))
+			continue
+		new /obj/effect/hallucination/tripper/spider_web(T, target)
+
+	new /obj/effect/hallucination/chaser/attacker/terror_spider(get_turf(vent), target)
+
+/obj/effect/hallucination/chaser/attacker/terror_spider
+	hallucination_icon = 'icons/mob/terrorspider.dmi'
+	hallucination_icon_state = "terror_green"
+	duration = 30 SECONDS
+	damage = 25
+
+/obj/effect/hallucination/chaser/attacker/terror_spider/Initialize(mapload, mob/living/carbon/target)
+	. = ..()
+	name = "Green Terror spider ([rand(100, 999)])"
+
+/obj/effect/hallucination/chaser/attacker/terror_spider/attack_effects()
+	do_attack_animation(target)
+	target.playsound_local(get_turf(src), 'sound/weapons/bite.ogg', 50, TRUE)
+	to_chat(target, "<span class='userdanger'>[name] bites you!</span>")
+
+/obj/effect/hallucination/chaser/attacker/terror_spider/on_knockdown()
+	target.visible_message("<span class='warning'>[target] recoils as if hit by something, before suddenly collapsing!</span>",
+						   "<span class='userdanger'>[name] bites you!</span>")
+
+/**
+  * # Hallucination - Spider Web
+  *
+  * A fake spider web that trips the target if crossed.
+  */
+/obj/effect/hallucination/tripper/spider_web
+	name = "spider web"
+	desc = "It's stringy and sticky."
+	hallucination_icon = 'icons/effects/effects.dmi'
+	hallucination_icon_state = "stickyweb1"
+	hallucination_override = TRUE
```

Comment:
`hallucination_override` makes the hallucination usable by clicking or hovering it. If you try to hover a hallucination with that setting off, it will have no mouse opacity whatsoever

Reply:
Alright, the ref I was looking at doesn't mention that. http://www.byond.com/docs/ref/#/atom/var/override